Packages/manifest.json

```diff
@@ -1,5 +1,8 @@
 {
   "dependencies": {
+    "com.unity.ide.rider": "1.2.1",
```

Comment:
@adrenak should these manifest changes be in here?

Reply:
Added it here because some customers could be on Rider IDE. Without this the IDE + Unity workflow doesn't work well.